### PR TITLE
CVSB-10218

### PR DESCRIPTION
**Pull request by vladddragomir**

### CODE REVIEW

File: src/pages/testing/test-creation/test-types-list/test-types-list.spec.ts [hunk 1]
```diff
@@ -56,6 +58,7 @@ describe('Component: TestTypesListPage', () => {
         {provide: FirebaseLogsService, useClass: FirebaseLogsServiceMock},
         {provide: TestTypeService, useClass: TestTypeServiceMock},
         {provide: VehicleService, useValue: vehicleServiceSpy},
+        {provide: AuthService, useClass: AuthServiceMock},
```
Comment (agoncearenco) [inline src/pages/testing/test-creation/test-types-list/test-types-list.spec.ts]: Why did you need this?

**Comment by vladddragomir** [comment]
i tried but had very hard time with it because the pipes are not using Test Bed. if you have any suggestion please feel free to share it with me

File: src/pages/testing/test-creation/test-types-list/test-types-list.spec.ts (hunk 1 above)
Comment (vladddragomir) [reply src/pages/testing/test-creation/test-types-list/test-types-list.spec.ts]: because i injected auth service in the pipe class and this class uses the pipe

Comment (agoncearenco) [reply src/pages/testing/test-creation/test-types-list/test-types-list.spec.ts]: Strange. And you do not need it inside the actual `test-types-list.ts` file?

Comment (vladddragomir) [reply src/pages/testing/test-creation/test-types-list/test-types-list.spec.ts]: it's needed because of the pipe

Comment (agoncearenco) [reply src/pages/testing/test-creation/test-types-list/test-types-list.spec.ts]: Strange but ok!

**Comment by agoncearenco** [comment]
Approving as testing this functionality appears to be too difficult.

File: src/pipes/filter-test-type-by-vehicle/filter-test-type-by-vehicle.pipe.ts [hunk 2]
```diff
@@ -1,12 +1,17 @@
...
+import { TEST_TYPES_IDS, TESTER_ROLES } from "../../app/app.enums";
 
 @Pipe({
   name: 'filterTestTypeByVehicle',
 })
-
 export class FilterTestTypeByVehiclePipe implements PipeTransform {
+
+  constructor(private authService: AuthService) {
+  }
```
Comment (KHodderDVSA) [inline src/pipes/filter-test-type-by-vehicle/filter-test-type-by-vehicle.pipe.ts]: implicit declaration is confusing. Looks like it's unused. Better to explicitly declare, even though it's longer.

Comment (vladddragomir) [reply src/pipes/filter-test-type-by-vehicle/filter-test-type-by-vehicle.pipe.ts]: well, it's like in the whole app

Comment (KHodderDVSA) [reply src/pipes/filter-test-type-by-vehicle/filter-test-type-by-vehicle.pipe.ts]: What's your point? Please declare it.

Comment (KHodderDVSA) [reply src/pipes/filter-test-type-by-vehicle/filter-test-type-by-vehicle.pipe.ts]: Angular docs support this style :/
